### ISSUE TITLE
Daemon Service will not start on Manjaro

### ISSUE THREAD

**Issue by GingerBobRoss** [ilya-zlobintsev/LACT#318]
### Checklist

- [X] The problem is not listed in the [hardware support matrix](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#hardware-support) as a known limitation
- [X] I've included a [debug snapshot](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#reporting-issues) if the issue is related to hardware configuration

### Bug description

I installed LACT on Manjaro running kernel 6.8 and after adding my username to the Config.yaml  in /etc/lact/config.yaml and restarting the daemon service  with sudo systemctl restart lactd   
 I still can not get LACT to enable with sudo systemctl enable --now lactd
running LACT in terminal reports  that the socket file is not found

![Screenshot_20240501_011626](https://github.com/ilya-zlobintsev/LACT/assets/51100139/7177319a-8f76-4e53-bbf3-d1bd7bd6080c)
![Screenshot_20240501_011613](https://github.com/ilya-zlobintsev/LACT/assets/51100139/773f6ce0-7bfb-4e6d-bed1-c5dfa8eb3e59)
![Screenshot_20240430_233407](https://github.com/ilya-zlobintsev/LACT/assets/51100139/b51b9787-7932-4120-9859-735452b1af33)
![Screenshot_20240501_011947](https://github.com/ilya-zlobintsev/LACT/assets/51100139/ff84e5c4-6382-4e26-9dd0-df6b254504fb)

[LACT-sysfs-snapshot-20240501-012044.tar.gz](https://github.com/ilya-zlobintsev/LACT/files/15174363/LACT-sysfs-snapshot-20240501-012044.tar.gz)

### System info

```markdown
- LACT version: 5.4
- GPU model:Sapphire Nitro+ RX 7900XT
- Kernel version:6.8.5-1
- Distribution: Manjaro
```

**Comment by ilya-zlobintsev** [comment]
Your config is incorrect. It should be
```
admin_groups:
- gonger
```
And you can check if the service is running with  `systemctl status lactd`

**Comment by GingerBobRoss** [comment]
That seems to have fixed it, but im still not able to make any adjustments due to the error below
Caused by:
0: Could not set default power profile mode
1: Got error from daemon, end of client boundary
2: Failed to edit GPU config and set power profile mode
3: Could not apply settings, and could not reset to default settings
4: Failed to set power cap: 150
5: io error: Invalid argument (os error 22)

The service is running now and i restarted the daemon as well and closed and reopened lact as well as rebooted.

**Comment by ilya-zlobintsev** [comment]
Try removing the `gpus` section in the config file (keep just the `gpus:` line, delete everything after) and restarting the service - it looks like you've already had some custom settings applied?

**Comment by GingerBobRoss** [comment]
Thanks!
I will try that when I get off work tonight and report back!

**Comment by GingerBobRoss** [comment]
I got that issue resolved, but i feel like its one issue after the other.
Ive noticed that my TDP usage spikes up to 450w when i have the power usage limit set to 235w and none of the clockspeed and undervolt options works
[LACT-sysfs-snapshot-20240502-205615.tar.gz](https://github.com/ilya-zlobintsev/LACT/files/15195548/LACT-sysfs-snapshot-20240502-205615.tar.gz)

**Comment by ilya-zlobintsev** [comment]
According to the snapshot, the power limit is set correctly, and there's a voltage offset configured at -60mV. So it doesn't seem like there's a problem on lact's side.

**Comment by GingerBobRoss** [comment]
> According to the snapshot, the power limit is set correctly, and there's a voltage offset configured at -60mV. So it doesn't seem like there's a problem on lact's side.


Sorry for the late response.
I got everything working with the exception  of having  fan control since thats an issue with Overdrive in the kernel.
Really hope AMD grants that  control soon. It still isnt fixed on kernel 6.9. 
having my fans turn off and then rapidly turn on is quite annoying.
